.travis.yml

```diff
@@ -33,11 +33,14 @@ install:
 script:
   - npm test
   - NODE_ENV=production node_modules/.bin/builder run build-static
+  - node_modules/.bin/builder run install-selenium
+  - ROWDY_SETTINGS=local.firefox node_modules/.bin/builder run test-func-static-prod
```

Comment:
@paulathevalley hmm, I'm not sure, I just set it up the same way that we had it in component-playground-docs. it's like this in victory-docs too